Core/src/com/basistech/df/cybertriage/autopsy/malwarescan/MalwareScanIngestModule.java

```diff
@@ -292,78 +433,397 @@ private void handleBatch(List<FileRecord> fileRecords) {
             }
 
             try {
-                // get an auth token with the license
-                AuthTokenResponse authTokenResponse = ctApiDAO.getAuthToken(licenseInfo.getDecryptedLicense());
+                List<CTCloudBean> repResult = getHashLookupResults(ingestJobState, md5Hashes);
+                handleLookupResults(ingestJobState, md5ToObjId, repResult);
+            } catch (Exception ex) {
+                notifyWarning(
+                        Bundle.MalwareScanIngestModule_SharedProcessing_generalProcessingError_title(),
+                        Bundle.MalwareScanIngestModule_SharedProcessing_generalProcessingError_desc(),
+                        ex);
+            }
+        }
+
+        /**
+         * Handles results received from CT Cloud.
+         *
+         * @param ingestJobState The current state of operations of the ingest
+         * module.
+         * @param md5ToObjId The mapping of md5 to a list of object ids.
+         * @param repResult The ct cloud results.
+         * @throws org.sleuthkit.datamodel.Blackboard.BlackboardException
+         * @throws TskCoreException
+         */
+        @Messages({
+            "MalwareScanIngestModule_SharedProcessing_exhaustedResultsHashLookups_title=Some Lookup Results Not Processed",
```

Comment:
We seem to have several copies of the same basic message. Can we reduce these? We shoudl be more precise. Something like "Not all files were processed".